File: model-bean/src/main/java/com/bnorm/barkeep/model/bean/IngredientBean.java
// Copyright 2016 (C) BNORM Software. All rights reserved.
package com.bnorm.barkeep.model.bean;

import javax.annotation.Nullable;

import com.bnorm.barkeep.model.Ingredient;

public class IngredientBean implements Ingredient {

  private Long id;
  private String title;
  private Ingredient parent;

  @Nullable
  @Override
  public Long getId() {
    return id;
  }

  public void setId(Long id) {
    this.id = id;
  }

  @Nullable
  @Override
  public String getTitle() {
    return title;
  }

  public void setTitle(String title) {
    this.title = title;
  }

  @Nullable
  @Override
  public Ingredient getParent() {
    return parent;
  }

  public void setParent(Ingredient parent) {
    this.parent = parent;
  }
}

File: model-bean/src/main/kotlin/com/bnorm/barkeep/model/bean/IngredientBean.kt
// Copyright 2016 (C) BNORM Software. All rights reserved.
package com.bnorm.barkeep.model.bean

import com.bnorm.barkeep.model.Ingredient

class IngredientBean : Ingredient {

  override var id: Long = -1
  override var title: String? = null
  override var parent: Ingredient? = null
}
